Formula/zlint.rb

```diff
@@ -0,0 +1,38 @@
+class Zlint < Formula
+  desc "X.509 Certificate Linter focused on Web PKI standards and requirements"
+  homepage "https://github.com/zmap/zlint"
+  url "https://github.com/zmap/zlint.git",
+      tag:      "v3.4.1",
+      revision: "84da589869bc2d096b7ae42606bf75c602d88806"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "--directory=v3", "GIT_VERSION=v#{version}", "zlint"
+    bin.install "v3/zlint"
+  end
+
+  test do
+    assert_match "ZLint version v#{version}",
+      shell_output("#{bin}/zlint -version")
+
+    cert = [
+      "-----BEGIN CERTIFICATE-----",
+      "MIIBzjCCAVSgAwIBAgIUUS5gI8Zv+vKmMsc4JnKpXoNUfXIwCgYIKoZIzj0EAwMw",
+      "HjEcMBoGA1UEAwwTaG9tZWJyZXctemxpbnQtdGVzdDAeFw0yMjAxMDIwNDAwMjNa",
+      "Fw0yMzAxMDIwNDAwMjNaMB4xHDAaBgNVBAMME2hvbWVicmV3LXpsaW50LXRlc3Qw",
+      "djAQBgcqhkjOPQIBBgUrgQQAIgNiAASn3DDorzrDQiWnnXv0uS722si9zx1HK6Va",
+      "CXQpHm/8t1SmMEYdVIU4j5UzbVKpoMIkk9twC3AiDUVZwdBNL2rqO8smZiKOh0Tz",
+      "BnRf8OBu55C7fsCHRayljjW0IpyZCjCjUzBRMB0GA1UdDgQWBBRIDxULqVXg4e4Z",
+      "+3QzKRG4UpfiFjAfBgNVHSMEGDAWgBRIDxULqVXg4e4Z+3QzKRG4UpfiFjAPBgNV",
+      "HRMBAf8EBTADAQH/MAoGCCqGSM49BAMDA2gAMGUCMQCCH1Sb8Hda7CttzpQjUb9c",
+      "VuG4qtyprIRvovHqrpRT0fYUJW9YmtUAHFTGijzwZSsCMAQXEXsFblfY4cwX3DYR",
+      "qg5yjd3Syn4cBMqtHKVYyi5s+6+vOONWPYBn9re7oF56jQ==",
+      "-----END CERTIFICATE-----",
+    ].join("\n") + "\n"
+    File.write("cert.pem", cert)
+
+    shell_output("#{bin}/zlint -longSummary cert.pem")
```

Comment:
```suggestion
    (testpath/"cert.pem").write <<~EOS
      -----BEGIN CERTIFICATE-----
      MIIBzjCCAVSgAwIBAgIUUS5gI8Zv+vKmMsc4JnKpXoNUfXIwCgYIKoZIzj0EAwMw
      HjEcMBoGA1UEAwwTaG9tZWJyZXctemxpbnQtdGVzdDAeFw0yMjAxMDIwNDAwMjNa
      Fw0yMzAxMDIwNDAwMjNaMB4xHDAaBgNVBAMME2hvbWVicmV3LXpsaW50LXRlc3Qw
      djAQBgcqhkjOPQIBBgUrgQQAIgNiAASn3DDorzrDQiWnnXv0uS722si9zx1HK6Va
      CXQpHm/8t1SmMEYdVIU4j5UzbVKpoMIkk9twC3AiDUVZwdBNL2rqO8smZiKOh0Tz
      BnRf8OBu55C7fsCHRayljjW0IpyZCjCjUzBRMB0GA1UdDgQWBBRIDxULqVXg4e4Z
      +3QzKRG4UpfiFjAfBgNVHSMEGDAWgBRIDxULqVXg4e4Z+3QzKRG4UpfiFjAPBgNV
      HRMBAf8EBTADAQH/MAoGCCqGSM49BAMDA2gAMGUCMQCCH1Sb8Hda7CttzpQjUb9c
      VuG4qtyprIRvovHqrpRT0fYUJW9YmtUAHFTGijzwZSsCMAQXEXsFblfY4cwX3DYR
      qg5yjd3Syn4cBMqtHKVYyi5s+6+vOONWPYBn9re7oF56jQ==
      -----END CERTIFICATE-----
    EOS
    out = shell_output("#{bin}/zlint -longSummary cert.pem")
    %w[
      e_ca_organization_name_missing
      e_root_ca_key_usage_present
      e_ca_country_name_missing
      e_ca_key_usage_missing
    ].each do |err|
      assert_match err, out
    end
```

Reply:
Applied this change with a slight variation on the cert used and lint results.